### DISCUSSION TITLE
Update Script to Support webcal links

### DISCUSSION THREAD

**Discussion by clhill** [derekantrican/GAS-ICS-Sync#332] [Ideas]
Does the script support webcal links? Can it be updated to include this if not? I added the script to an account using webcal links and the calendars never show up.

Appreciate any help any one can provide

**Comment by derekantrican** [comment]
Yes - we replace `webcal://` with `https://` so we can properly get the calendar data, but you can provide a webcal link: https://github.com/derekantrican/GAS-ICS-Sync/blob/master/Helpers.gs#L77

Are you willing to provide the webcal link you're using?

**Comment by clhill** [comment]
Thanks for the info. Changing the text of webcal:// to https:// worked great. All calendars were imported and are syncing at the desired refresh rate.

This app is great. Thank you guys for the work on this.
